src/System.Private.ServiceModel/tests/Common/Infrastructure/OSHelper.cs

```diff
@@ -147,8 +142,8 @@ private static OSID DetectCurrentOS()
             return osid;
         }
 
-        // Maps from the $(TestNugetRuntimeId) property to corresponding OSID.
-        // Returns OSID.None if cannot determine the OSID.
+        // The $(TestNugetRuntimeId) property was deprecated in build tools v2.
+        // No longer calling this method.
         private static OSID OSIDfromTestRuntime()
```

Comment:
Let's clean this up too so we don't cause confusion for the future 😄

Reply:
Done